README.md

```diff
@@ -202,7 +202,7 @@ sudo make install
 #### macOS (using brew)
 
 ```bash
-brew install curl cmake
+brew install curl libressl c-ares
```

Comment:
cmake is still needed. I hit this yesterday, I used `brew install openssl`, but from a brief read `libressl` is better supported at least for mac IIUC. Cool, I'll try this instead.

Reply:
Fixed.